sotopia/cli/benchmark/benchmark.py

```diff
@@ -262,9 +263,57 @@ def _set_up_logs(
     )
 
 
+def save_to_jsonl(
+    model_rewards_dict: Dict[str, Dict[str, float]],
+    partner_model: str,
+) -> None:
+    simplified_model_name = partner_model.split("/")[-1]
+    output_fn = f"./models_vs_{simplified_model_name}.jsonl"
+    outputs: List[str] = []
+    for model, rewards in model_rewards_dict.items():
+        formatted_reward = OrderedDict(
+            {
+                "model_name": model,
+                **{
+                    f"{v[0]} {v[1]}": rewards[k]
+                    for k, v in dimension_range_mapping.items()
```

Comment:
Why converting to the acronyms and printing out the range is essential here?

Reply:
Here I am going to output the results to the file so that it can be parsed, and the sample is here https://github.com/sotopia-lab/sotopia-space/blob/main/data_dir/models_vs_gpt35.jsonl.
Actually I am thinking of providing a command line table output but pandas is not in current Sotopia requirements. Do we have to include it? Or there are other ways to output in a formatted way?

Reply:
Rich as our default CLI library provides a structured output

https://rich.readthedocs.io/en/stable/tables.html

Reply:
Okay, I got it. Yeah, if we want to make the output consistent with the sotopia-space format that sounds good to me.